### PR TITLE
Completed prep-javascript-dom-manipulation exercise

### PR DESCRIPTION
**Pull request by Matthew-Abebe**
prep-javascript-dom-manipulation

### CODE REVIEW

File: prep-javascript-dom-manipulation/index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  </style>
+</head>
+
+<body>
+  <div id="container">
+    <p class="first-paragraph">
+      I really like coding.
+    </p>
+    <p class="second-paragraph">
+      'I also really like to watch television and eating pancakes.'
```
Comment (thebearingedge) [inline prep-javascript-dom-manipulation/index.html:21]: Try adding this text with your JavaScript code.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Nice! Just one small change

**Comment by Matthew-Abebe** [comment]
prep-javascript-dom-creation-screenshot (2)

**Review by thebearingedge — APPROVED** [review]
👍

**Pull request merged**